ext/scheduler/airflow2/resources/expected_compiled_template.py

```diff
@@ -220,4 +223,4 @@
 
 # arrange failure hook after post hooks
 
-hook_predator >> [ hook_hook__dash__for__dash__fail,]
\ No newline at end of file
+hook_predator >> [ hook_hook__dash__for__dash__fail,]
```

Comment:
@novanxyz also remove the additional new line at the end of this expected file. thanks!